packages/preview/pinit/0.1.0/typst.toml

```
@@ -0,0 +1,10 @@
+[package]
+name = "pinit"
+version = "0.1.0"
+authors = ["OrangeX4"]
+entrypoint = "lib.typ"
+exclude = ["./examples", "./README.md"]
```

```suggestion
exclude = ["./examples"]
```
Some existing packages exclude the README, but the key was only intended for large support files.

---

OK, fixed it.